README.md

```diff
@@ -220,7 +219,7 @@ RCT_EXPORT_METHOD(bindVideoView:(NSNumber * _Nonnull)viewIdentifier tileId:(NSNu
 {
   dispatch_async(dispatch_get_main_queue(), ^{
     UIView* view = [self.bridge.uiManager viewForReactTag:viewIdentifier];
-    [meetingSession.audioVideo bindVideoViewWithVideoView:(RNVideoView*)view tileId:[tileId integerValue]];
+    [meetingSession.audioVideo bindVideoViewWithVideoView:(DefaultVideoRenderView*)view tileId:[tileId integerValue]];
```

Comment:
Why does this need to change? I think it still exposes [RNVideoView](https://github.com/aws-samples/amazon-chime-react-native-demo/blob/master/ios/RNDemo/RNVideoViewManager.m#L11)

Reply:
Nvm it is readme. Thought this was JS layer.

Reply:
We expose `RNVideoView` to RN, biut native iOS code needs to reference and cast to DefaultVideoRenderView. This is part of your code change in #40